byterun/sys.c

```diff
@@ -49,7 +50,6 @@
 #include "caml/debugger.h"
 #include "caml/fail.h"
 #include "caml/gc_ctrl.h"
-#include "caml/instruct.h"
```

Comment:
This include didn't seem to be serving any purpose in this file, but it is a nuisance when `<windows.h>` is included, because Microsoft also define `ATOM`